identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -106,3 +105,5 @@ <h2 class="form__heading">Formatting</h2>
     </div>
     <div class="manage-account__modalBg js-manage-account__modalCloser"></div>
 </div>
+
+</form>
```

Comment:
Indention looks messed up now. Is it only the diff?

Reply:
i wanted to keep the diff on that commit tiny, then make a second one just fixing the indent, then i forgot about making the second one ✌️